Conch/lib/Conch/Route/Orc.pm

```diff
@@ -0,0 +1,94 @@
+=head1 NAME
+
+Conch::Route::Orc
+
+=head1 DESCRIPTION
+
+Mojo routes for Conch's orchestration system
+
+=head1 SYNOPSIS
+
+Conch::Route::Orc->load( $r );
+
+=head1 METHODS
+
+=cut
+
+package Conch::Route::Orc;
+
+use Mojo::Base -base, -signatures;
+
+
+=head2 load
+
+Load up all the routes and attendant subsystems
+
+=cut
+
+use Conch::Orc;
+use Conch::Controller::Orc::Workflows;
+use Conch::Controller::Orc::WorkflowExecutions;
+use Conch::Controller::Orc::Lifecycles;
+
+sub load ( $class, $r ) {
+	my $o = $r->under("/o");
+
+	my $ls = $o->under("/lifecycles");
```

Comment:
For all other resources, we've used singular nouns, with `/noun` providing the group and `/noun/:id` providing the singular. I'd prefer to keep this consistent throughout the application. Routes like `/workspaces` but `/workspace/:id` are especially meddlesome, especially when we don't have this pattern elsewhere.

I see that `/executions/(active|stopped|completed)` doesn't fit the group-id model and may have motivated the change. If it doesn't work to make it singular or even use query parameters, this exception is fine.